clippy_lints/src/formatting.rs

```diff
@@ -214,7 +214,7 @@ fn check_else(cx: &EarlyContext<'_>, expr: &Expr) {
         // the snippet should look like " else \n    " with maybe comments anywhere
         // it’s bad when there is a ‘\n’ after the “else”
         && let Some(else_snippet) = snippet_opt(cx, else_span)
-        && let Some((pre_else, post_else)) = else_snippet.split_once("else")
```

Comment:
Are there any situations where the lint should fire and there are comments? I feel like `&& !else_snippet.contains('/')` would be enough

Reply:
I agree that `&& !else_snippet.contains('/')` is a smart fix. I made an adjustment!